connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/OracleIntegrationSuite.scala

```diff
@@ -88,6 +89,11 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JDBCTes
       s"jdbc:oracle:thin:system/$oracle_password@//$ip:$port/freepdb1"
   }
 
+  override val defaultMetadata = new MetadataBuilder()
```

Comment:
why is this change?

Reply:
The superclass tests with strings. As we use varchar to map string values for Oracle, it's now a special case.